src/wizard/boards.ts

```diff
@@ -1,22 +1,26 @@
 import { html } from "lit";
 import { SupportedBoards } from "../api/boards";
 
-export const boardSelectOptions = (boards: SupportedBoards | undefined) =>
+export const boardSelectOptions = (
+  boards: SupportedBoards | undefined,
+  defaultBoard: string | null = null
+) =>
   boards &&
-  html`
-    <optgroup label="ESP32">
-      ${Object.keys(boards.esp32).map(
-        (key) => html`<option value="${key}">${boards.esp32[key]}</option>`
+  Object.values(boards).map((group) => {
+    const hasDefault =
```

Comment:
This is weird. You're going to lookup the default and render it per group? It should always be on top of the whole list.

Reply:
Made it lookup the default once in all platforms.